Tools/CISupport/ews-build/steps.py

```diff
@@ -5929,6 +5930,83 @@ def getResultSummary(self):
             configuration += ', Xcode: {}'.format(xcode_version)
         return {'step': configuration}
 
+class CheckOutLLVMProject(git.Git, AddToLogMixin):
+    name = 'checkout-llvm-project'
+    directory = 'llvm-project'
+    branch = 'webkit'
+    CHECKOUT_DELAY_AND_MAX_RETRIES_PAIR = (0, 2)
+    GIT_HASH_LENGTH = 40
+    haltOnFailure = False
+
+    def __init__(self, **kwargs):
+        repourl = f'{GITHUB_URL}/rniwa/llvm-project.git'
+        super().__init__(
+            repourl=repourl,
+            workdir=self.directory,
+            retry=self.CHECKOUT_DELAY_AND_MAX_RETRIES_PAIR,
+            timeout=2 * 60 * 60,
```

Comment:
please verify if it could take 2 hours to checkout this repo.

Reply:
Oops! Should be up to 3 minutes.